Extending-OSVR/ConfiguringHDKViveTracking.md

```diff
@@ -0,0 +1,260 @@
+# Configuring OSVR HDK2 with Vive Puck Tracking
```

Comment:
I'd name the file `ConfiguringHDKLighthouseTracking.md` instead of `-ViveTracking`.